Tools/machines/frontier-olcf/submit.sh

```diff
@@ -6,12 +6,155 @@
 #SBATCH -t 00:10:00
 #SBATCH -p batch
 #SBATCH --ntasks-per-node=8
-# Due to Frontier's Low-Noise Mode Layout only 7 instead of 8 cores are available per process
+# Due to Frontier's Low-Noise Mode Layout only
+# 7 instead of 8 cores are available per process
 # https://docs.olcf.ornl.gov/systems/frontier_user_guide.html#low-noise-mode-layout
 #SBATCH --cpus-per-task=7
 #SBATCH --gpus-per-task=1
 #SBATCH --gpu-bind=closest
 #SBATCH -N 20
+# In order to broadcast the WarpX executable and
+# the linked libraries to each node we need to
+# use the Non-Volatile Memory (NVMe) on node storage devices
+# https://docs.olcf.ornl.gov/systems/frontier_user_guide.html#sbcasting-a-binary-with-libraries-stored-on-shared-file-systems
+#SBATCH -C nvme
+
+# In case you are using pywarpx, please uncomment the following line.
+# This is needed to select the correct strategy to broadcast the
+# warpx executable and the associated libraries to all the nodes.
+#is_pywarpx=true
+
+if [ $is_pywarpx ]; then
```

Comment:
[Clarity](https://github.com/azet/community_bash_style_guide):
```suggestion
if [ ${is_pywarpx} ]; then
```